torch/utils/_triton.py

```diff
@@ -0,0 +1,17 @@
+import functools
+
+import torch
+
+from torch._inductor.cuda_properties import get_device_capability
+
+
+@functools.lru_cache(None)
+def has_triton() -> bool:
```

Comment:
IMO function name is wrong, it should be something like `has_trition_support`, because it also checks that GPU capable for running triton generated code is present

Reply:
I just moved the code from one location to another, we can follow up with a rename if needed.